Source/NSubstitute.2010.sln.DotSettings

```diff
@@ -0,0 +1,4 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
>...it might be useful to have a .DotSettings file in the repo so that anyone else contributing that uses ReSharper a very common tool, use the same coding conventions.

Agreed. But that's a separate issue and PR. And we must agree with the conventions :smile: